lib/commands/implementations/schedule.js

```diff
@@ -0,0 +1,10 @@
+const moment = require('moment');
+const Command = require('../command-interface');
+const CommandOutput = require('../command-output');
+
+function getNextStreamDate(input, services) {
+  return services.schedule.findNextStreamDay()
+    .then(nextStreamDay => new CommandOutput(null, `"Stream" scheduled to begin in 14h 11m ${moment(nextStreamDay.dateTime).fromNow()} https://destiny.gg/schedule`));
```

Comment:
Looks like there's some placeholder text left in the output!

Reply:
Also, throw a .catch(err => )

On here that will just output that the command failed, and then logs the error with the logger.